From 1a6d915a4eca71f0506e2dc5a7bf0fcae9fc15a1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ville=20Syrj=C3=A4l=C3=A4?= <ville.syrjala@linux.intel.com>
Date: Tue, 30 Jun 2015 15:33:52 +0300
Subject: [PATCH 0825/4706] drm/i915: Bump HDMI min port clock to 25 MHz
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit 6fd765d0591a79934b268137fc7c0fbfa0e2e3b4 upstream

Increase the HDMI port minimum port clock from 20 to 25 MHz. This is
is the minimum listed in the DVI/HDMI specs, and it's also the
documented minimum DPLL frequency for most of our platforms.

Signed-off-by: Ville Syrjälä <ville.syrjala@linux.intel.com>
Reviewed-and-tested-by: Imre Deak <imre.deak@intel.com>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
---
 drivers/gpu/drm/i915/intel_hdmi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_hdmi.c b/drivers/gpu/drm/i915/intel_hdmi.c
index 00c4b40..69244ed 100644
--- a/drivers/gpu/drm/i915/intel_hdmi.c
+++ b/drivers/gpu/drm/i915/intel_hdmi.c
@@ -1175,7 +1175,7 @@ intel_hdmi_mode_valid(struct drm_connector *connector,
 	if (clock > hdmi_portclock_limit(intel_attached_hdmi(connector),
 					 true))
 		return MODE_CLOCK_HIGH;
-	if (clock < 20000)
+	if (clock < 25000)
 		return MODE_CLOCK_LOW;
 
 	if (mode->flags & DRM_MODE_FLAG_DBLSCAN)
-- 
1.7.5.4

